legal-api/src/legal_api/reports/report.py

```diff
@@ -216,6 +223,24 @@ def _format_directors(self, directors):
                 pass
         return directors
 
+    def _set_translations(self, filing):
+        # initial (existing) translations, if any
+        nameTranslations = filing['business'].get('nameTranslations', [])
```

Comment:
@lekshmimallika-aot I'm assuming there will be a list of existing name translations in the business object. Is this actually the plan? If not, where do I get the initial list of name translations for an alteration?